src/cavendish_particle_tracks/_main_widget.py

```diff
@@ -354,7 +332,6 @@ def set_UI_image_loaded(
         else:
             # Set margins (left, top, right, bottom)
             self.buttonbox.setContentsMargins(200, 0, 200, 0)
```

Comment:
If you go for just disabling the buttons, you probably want to get rid of this modifications of the margins too.

Reply:
Oh, sure. Let me test. Again: the widget _shouldn't_ be in the business of dictating it's own size, if we can about it. That's the responsibility of `napari`.